### PR TITLE
Feature/readd callback logic

### PR DESCRIPTION
**Pull request by roesslerj**
Revert d291761 (#855): Remove Callback Service sources and references in Dockerfiles/POMs

### CODE REVIEW

**Review by hilmarf — DISMISSED** [review]

File: services/callback/src/main/resources/application.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+      TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+      TLS_DHE_DSS_WITH_AES_128_GCM_SHA256
+      TLS_DHE_DSS_WITH_AES_256_GCM_SHA384
+      TLS_DHE_RSA_WITH_AES_128_GCM_SHA256
+      TLS_DHE_RSA_WITH_AES_256_GCM_SHA384
+      TLS_AES_128_GCM_SHA256
+      TLS_AES_256_GCM_SHA384
+      TLS_AES_128_CCM_SHA256
+    key-password: ${SSL_SUBMISSION_KEYSTORE_PASSWORD}
```
Comment (FelixRottler) [inline services/callback/src/main/resources/application.yaml:75]: Is it correct to use submission here or should there also be a CALLBACK_KEYSTORE_PATH?

File: services/callback/src/main/resources/bootstrap-cloud.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  cloud:
+    vault:
+      enabled: true
+      generic:
+        enabled: false
+      kv:
+        enabled: true
+        backend: ${VAULT_BACKEND}
+        profile-separator: '/'
+        application-name: 'submission'
```
Comment (FelixRottler) [inline services/callback/src/main/resources/bootstrap-cloud.yaml:14]: Same here

Comment (roesslerj) [reply services/callback/src/main/resources/bootstrap-cloud.yaml:14]: Good catch!

File: services/callback/src/main/resources/application.yaml (hunk 1 above)
Comment (roesslerj) [reply services/callback/src/main/resources/application.yaml:75]: Good catch, thanks!

**Review by FelixRottler — APPROVED** [review]

**Review by hilmarf — APPROVED** [review]

**Pull request merged**